.vsts-pipelines/templates/steps/script.yml

```diff
@@ -0,0 +1,70 @@
+# File: templates/steps/script.yml
+
+parameters:
+  name: 'DO NOT LEAVE DEFAULT'
+  agentOs: 'Windows'
+  beforeBuild: []
+  buildScript: ''
+  buildArgs: ''
+  afterBuild: []
+  hasTests: false
+  autoArtifact: true
+  envVars:
```

Comment:
Is the behavior you want that if someone provides this parameter and explicitly sets envVars, then the rest of the envVars are cleared or that parameters passed are additive to what is defaulted here?

https://github.com/Microsoft/vsts-agent/blob/master/docs/preview/yamlgettingstarted-templates.md#example-insert-into-a-dictionary